src/java.base/share/classes/java/io/BufferedInputStream.java

```diff
@@ -394,7 +394,7 @@ public synchronized long skip(long n) throws IOException {
      * many bytes will not block, but may read or skip fewer bytes.
      * <p>
      * This method returns the sum of the number of bytes remaining to be read in
-     * the buffer (<code>count&nbsp;- pos</code>) and the result of calling the
+     * the buffer ({@code count - pos}) and the result of calling the
      * {@link java.io.FilterInputStream#in in}.available().
```

Comment:
I believe this should probably preferably be:

` * {@link java.io.FilterInputStream#in in}{@code .available()}.`

since `.available()` is obviously code.

Reply:
I agree that would probably be better. Will change before integrating.